src/content/app/tools/blast/components/blast-input-sequences/BlastInputSequencesHeader.tsx

```diff
@@ -102,4 +101,28 @@ const AddAnotherSequence = (props: AddAnotherSequenceProps) => {
   );
 };
 
+const SequenceSwitcher = (props: {
+  sequenceType: string;
+  onChange: (sequenceType: SequenceType) => void;
+}) => {
+  const options: { label: string; value: SequenceType }[] = [
+    {
+      label: 'Nucleotide',
+      value: 'dna'
+    },
+    {
+      label: 'Protein',
+      value: 'protein'
+    }
+  ];
+  return (
+    <RadioGroup
+      options={options}
+      direction="row"
+      selectedOption={props.sequenceType}
+      onChange={(val) => props.onChange(val as unknown as SequenceType)}
```

Comment:
Can it just be `as SequenceType`?

Reply:
It can. Weird, I was sure that when I tried it before, typescript complained of something or other.

✅